readme.md

```diff
@@ -69,15 +69,15 @@ Once installation is working, try creating and requesting resources. Note that t
 
 * **Question:** Describe what the above code is adding to your schema.
 
-* **Your Answer:**
+* **Your Answer:** This line of code will add a property `_id` into our model and the value of it will be automatically generated by mongoose. It will be a unique id for each document entry.
```

Comment:
This is not quite right. If you include the above in your code, it will require you to send an ObjectID when creating a new record.